### DISCUSSION TITLE
Nutanix Sync

### DISCUSSION THREAD

**Discussion by monteroman** [bb-Ricardo/netbox-sync#420] [Ideas]
I don't know if anyone's thought of looking into this for syncing VMs, but Nutanix has a decent API to pull info from and it would be great if this application could also sync Nutanix VMs and Clusters into Netbox.

**Comment by bb-Ricardo** [comment]
Hi, sorry I missed the discussions.

There are no plans to ingrate Nutanix. Maybe netbox-ssot can help here.
